main.go

```diff
@@ -33,6 +37,22 @@ func main() {
 		logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
 	}
 
+	var sentryHook *logrus_sentry.SentryHook
+	if *sentry != "" {
+		sentryLevels := []logrus.Level{
+			logrus.PanicLevel,
+			logrus.FatalLevel,
+			logrus.ErrorLevel,
+		}
+		sh, err := logrus_sentry.NewSentryHook(*sentry, sentryLevels)
+		if err != nil {
+			logrus.Warningf("Could not init sentry logger: %s", err)
```

Comment:
Let's not ignore the error here. if the user meant to add a Sentry hook (as evidenced by the fact that they provided a Sentry DSN), then we should not ignore errors doing that